internal/workflow/processing.go

```diff
@@ -738,7 +738,7 @@ func (w *ProcessingWorkflow) transferAM(sessCtx temporalsdk_workflow.Context, ti
 	// Delete transfer.
 	activityOpts = withActivityOptsForRequest(sessCtx)
 	err = temporalsdk_workflow.ExecuteActivity(activityOpts, am.DeleteTransferActivityName, am.DeleteTransferActivityParams{
-		Destination: uploadResult.RemotePath,
+		Destination: uploadResult.RemoteFilename,
```

Comment:
This is probably not a good idea. There's the problem here, the filename might be inside a child directory of the the remote directory instead of at the root which it currently is. We probably will add a child directory in the future to avoid something like name conflicts(though you could also just do this with a uuid for filenames) Either way just using the base here makes more sense in the workflow rather than the activity as it require far less changes.